shell/platform/android/image_external_texture.h

```diff
@@ -49,6 +56,16 @@ class ImageExternalTexture : public flutter::Texture {
   virtual void Detach() = 0;
   virtual void ProcessFrame(PaintContext& context, const SkRect& bounds) = 0;
 
+  sk_sp<flutter::DlImage> FindImage(uint64_t key);
+  void UpdateKey(const sk_sp<flutter::DlImage>& image, uint64_t key);
+  uint64_t AddImage(const sk_sp<flutter::DlImage>& image, uint64_t key);
+  void Clear();
+
+  struct LRUImage {
+    uint64_t key;
```

Comment:
uint64_t key = 0;

and then you don't need to explicitly initialize all the values below on lines 83-84

Reply:
Done